### PR TITLE
feat: HMA streaming indicators

### PR DESCRIPTION
**Pull request by DaveSkender**
adds Buffer list and StreamHub style indicators for Hull Moving Average

### CODE REVIEW

**Comment by DaveSkender** [comment]
@coderabbitai generate docstrings

File: .github/copilot-instructions.md [hunk 1]
```diff
@@ -1,75 +1,88 @@
...
 
 ### Code quality standards
 
-- All public methods must have XML documentation
-- Unit test coverage for all code paths
-- Performance tests for computationally intensive indicators
-- Validation for all user inputs
-- Consistent formatting using `.editorconfig`
+- Provide XML comments for all public types and members.
+- Cover happy paths, edge cases, and streaming flows with unit tests.
```
[the remark this answers is not in the document]
Comment (DaveSkender) [reply .github/copilot-instructions.md:55]: Our rules about using `decimal` are more nuanced than perhaps we were stating.
I've attempted to update the documentation in b331619aac97069d7526613ad22a49389a919f1e to clarify.

File: src/e-k/Hma/Hma.BufferList.cs [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        double wma = 0;
+        double[] values = buffer.ToArray();
+
+        for (int j = 0; j < periods; j++)
+        {
+            wma += values[j] * (j + 1) / divisor;
+        }
+
+        return wma;
+    }
```
[the remark this answers is not in the document]
Comment (DaveSkender) [reply src/e-k/Hma/Hma.BufferList.cs]: This is fairly normal since we standardize and sometimes use `decimal` precision in other indicators.  You can see the standard `HmaResult` returns `double`, so there's no issue here.

**Comment by DaveSkender** [comment]
@coderabbitai close your feedback items if they've been resolved
